test/app/presenter/ChessBoardPresenter.spec.ts

```diff
@@ -1,26 +1,61 @@
 import '@testing-library/jest-dom';
 import { ChessBoardView } from '../../../src/app/view/ChessBoardView';
-import { ChessModel } from '../../../src/app/model/ChessModel';
+import { ChessModel, PIECES_START_POSITION, Square } from '../../../src/app/model';
 import { ChessBoardPresenter } from '../../../src/app/presenter/ChessBoardPresenter';
 import { ViewEventBus } from '../../../src/app/view/events/ViewEventBus';
 import { ViewEvent } from '../../../src/app/view/events/ViewEvent';
 import { SquareWasClicked } from '../../../src/app/view/events/SquareWasClicked';
 import { InMemoryViewEventBus } from '../../../src/app/view/events/InMemoryViewEventBus';
-import { PIECES_START_POSITION } from '../../../src/app/model/Constances';
 
 describe('ChessBoardPresenter', () => {
-  const viewEvents: ViewEventBus = new InMemoryViewEventBus();
-  const view: ChessBoardView = chessBoardViewMock(viewEvents);
-  const model: ChessModel = { squaresWithPiece: PIECES_START_POSITION, move: jest.fn() };
-  const presenter: ChessBoardPresenter = new ChessBoardPresenter(view, model);
+  it('when square A1 was clicked on the view, then selected piece should be shown with its possible moves', () => {
+    const viewEvents: ViewEventBus = new InMemoryViewEventBus();
```

Comment:
Tutaj w Given masz duzo powtorzen, ktore roznia sie tylko possibleMoves, dobrze byloby to wyciagnac do metody i sparametryzowac tylko zmienna czescia.